app/controllers/api/v1/items_controller.rb

```diff
@@ -1,9 +1,38 @@
 class Api::V1::ItemsController < ApplicationController
+  rescue_from ActiveRecord::RecordInvalid, with: :missing_attributes
```

Comment:
I moved the rescue I made into ApplicationController, just to keep the other controllers as light as possible.